jnigen/lib/src/tools/build_summarizer.dart

```diff
@@ -29,11 +29,11 @@ Future<void> buildApiSummarizer() async {
   final pom = pkg.resolve('java/pom.xml');
   await Directory(toolPath).create(recursive: true);
   final mvnArgs = [
+    'package',
```

Comment:
`package` or `compile` should not make much difference here, because every merged PR will have it's Java tests passed in CI.

Reply:
Interesting, that's good to know about the CI testing, I'm happy with either `package` or `compile`.

If we don't need to run the Java tests due to CI, then maybe `compile` is better because on my machine `compile` is faster taking about 10s whereas `package` took about 17s. 

I'll make a commit which uses `compile` instead of `package` as it looks like the best option because it is faster and we don't need to run the tests.